source/manual/copying-data-locally.html.md

```diff
@@ -0,0 +1,63 @@
+---
+owner_slack: "#2ndline"
+title: Copying application data locally for development
```

Comment:
Since we often refer to this as "replicating" maybe "Replicate application data locally for development" will make it easier to find?